core-scala/src/test/scala/com/baeldung/scala/absClassAndtraits/AbstractClassAndTraitsTogetherTest.scala

```diff
@@ -0,0 +1,67 @@
+package com.baeldung.scala.absClassAndtraits
+
+import com.baeldung.scala.absClassAndtraits.TutorialHelper.{BirdLikeEmployee, Employee, SingerAndFlierEmployee, SuperWomanLikeEmployee}
+import org.scalatest.{Matchers, WordSpec}
+
+class AbstractClassAndTraitsTogetherTest extends WordSpec with Matchers {
+
+  val employeeFromDublin =
+    new Employee(1001,"Robert McCullam",50,"Dublin","Cork",12)
+
+  val employeeFromBucharest =
+    new Employee(1002,"Giorghe Hagi",60,"Bucharest","HQ",15)
+
+  val employeeFromCalcutta =
+    new Employee(1003,"Amartya Sen",60,"Calcutta","London",9)
+
+  val flierEmployeeFromDublin =
+    new BirdLikeEmployee(1004,"Paul Crowley",50,"Dublin","Cork",12)
+
+  val superWomandFromCalcutta =
+    new SuperWomanLikeEmployee(1005,"Tamanna Sengupta",60,"Calcutta","London",9)
+
+  val manySkilledEmployeeFromMunich =
+    new SingerAndFlierEmployee(1006,"Lothar Matheius",55,"Munich","Berlin",9)
+
+
+  "An employee aged over 60 and working for more than 10 years " should {
+    "be eligible for special allowance " in {
+      employeeFromBucharest.isEligibleForSpecialAllowance shouldBe  true
+    }
+  }
+
+  "An employee aged over 60 and working for less than 10 years " should {
+    "not be eligible for special allowance " in {
+      employeeFromCalcutta.isEligibleForSpecialAllowance shouldBe false
+    }
+  }
+
+  "A flyer employee from Dublin " should {
+    "have only 2 wings and fly normally " in {
+        flierEmployeeFromDublin.minWings               shouldBe 2
```

Comment:
There should be only 1 space between "flierEmployeeFromDublin.minWings" and "should've"

Reply:
Point taken, Martin.

Reply:
@glmartin By mistake, the check-in had been from my work account. Apologies. 

I am checking in again, from Baeldung account.